src/crimson/osd/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ add_executable(crimson-osd
   ${PROJECT_SOURCE_DIR}/src/osd/PGLog.cc
   ${PROJECT_SOURCE_DIR}/src/osd/osd_perf_counters.cc
   )
+target_compile_definitions(crimson-osd PRIVATE -DWITH_SEASTAR -DWITH_BLUESTORE)
```

Comment:
if you always enable `WITH_BLUESTORE`, why bother checking it in `src/crimson/os/CMakeLists.txt` ?

also, `-DWITH_SEASTAR` will be added when linked against libcrimson as it's part of the public interface of `crimson` exposed via `crimson::cflags`. so there is no need to specify it. am i missing anything?